dspace/config/modules/discovery.cfg

```diff
@@ -24,6 +24,13 @@ discovery.search.server = ${solr.server}/${solr.multicorePrefix}search
 # discovery.index.ignore-authority = false
 discovery.index.projection=dc.title,dc.contributor.*,dc.date.issued
 
+# Restricts the indexing of the submitter for archived items
+# By default the submitter information from the corresponding eperson is indexed
+# discovery.index.item.submitter.enabled = false
+# The submitter name can be replaced by some textual placeholder value
+# By default the original submitter name of the eperson is used
+# discovery.index.item.submitter.name = placeholder
```

Comment:
In today's Developer Meeting, we discussed this PR as a team. We were unclear why this option is needed, as it doesn't seem like it would provide improved behavior in the user interface.  But, we were not sure if we understand the use case.

Basically, what we assume will happen is the following:
1.  When `discovery.index.item.submitter.enabled = false` is set, in the MyDSpace Workflow tasks, faceting by submitter will result in the submitter facet being dropped (as no submitters exist).  This means this facet just won't appear.
2. When `discovery.index.item.submitter.name = placeholder` is set,  in the MyDSpace Workflow tasks, faceting by submitter will result in **all items appearing under that placeholder**.  It's unclear why anyone would want that behavior, as it seems like it's better for the facet to not appear.

We were not sure if we fully understand the use case for having this placeholder, or if it's better to simply remove it and only have the "enabled" option.

Reply:
@tdonohue Thank you for the feedback. You are right, by the current discovery configuration the submitter name will never appear on the UI or aggregate all items under that placeholder and the placeholder is useless. Thus I'v removed the placeholder code and simplified the code and tests.